code/bignum/Makefile

```diff
@@ -7,15 +7,16 @@ all: dist/bignum-test.exe
 # Defines rules for producing .checked, .krml, .depend, etc.
 include ../../Makefile.local
 
-CFLAGS += -I ../../../lib/c -I ../../lib/c -march=native -mtune=native -O3
+CFLAGS += -I ../../../lib/c -I ../../lib/c -march=native -mtune=native -O3 -DCOMPILE_INTRINSICS
```

Comment:
Is this really needed? If yes, does someone need to update all the build systems for all products currently using HACL* ?

Reply:
As long as they run the configure script, no update is required.
We don't run ./configure for local makefiles, that's why this change is needed.
[This](https://github.com/project-everest/hacl-star/commit/57c68278a4574038956f5b64b53ca8dd42abbd50) commit has changed the default behavior.

Reply:
Nobody runs the configure file.

Reply:
So the question is: what will break if this flag is not passed?

Reply:
It will take a fallback implementation or complain that the Hacl_IntTypes_Intrinsics.c file is not found.